[Technical Review] Add folder for WebDriver BiDi and create initial landing pages (#43153)
mdn/content: files/en-us/web/webdriver/reference/bidi/modules/index.md

---
title: WebDriver BiDi modules
short-title: Modules
slug: Web/WebDriver/Reference/BiDi/Modules
page-type: listing-page
browser-compat: webdriver.bidi
sidebar: webdriver
---

The WebDriver BiDi protocol is organized into modules.

> Each module represents a collection of related commands and events pertaining to a specific aspect of browser automation.

Reviewer: ```suggestion
Each module represents a collection of related commands and events used in specific browser automation cases.
```
I would make it sound a little simpler.
